CC feedback on DOCSP-27919 (#80)
mongodb/docs-csharp: source/fundamentals/connection/connect.txt

Connect to Atlas
~~~~~~~~~~~~~~~~

To connect to an Atlas deployment, you need to create a client. You can
create a client that uses your connection string and other 
client options by passing a ``MongoClientSettings`` object to the ``MongoClient``
constructor. To specify your connection URI, pass it to the ``FromConnectionString()``
method, creating a ``MongoClientSettings`` instance. To specify any other

> settings, set the relevant fields of the settings object.

Reviewer: Suggestion:
I think it would be easier to read if the terminology was consistent (client options vs settings). I think in this case, it's more confusing to make the switch in terminology because the class name is MongoClientSettings.

```suggestion
client options, set the relevant fields of the MongoClientSettings object.
```

Author: changed all references to options to settings

> 

Author: Yes, it was getting a little dense with all the monospace object references

To learn more about connection options, see the
:ref:`Connection Options page <csharp-connection-options>`.

> You can set the {+stable-api+} version as a setting to avoid

Reviewer: Suggestion:

I think using the verb "set" to describe an action on a "setting" can be confusing as there are other equivalent words. I think the current wording also attributes the "set" action to "as a setting" which is an incorrect relation and should be "the setting" instead.

Per another comment in this review, it could be helpful to stay consistent in terminology.

```suggestion
You can set the {+stable-api+} version client option to avoid
```

breaking changes when you upgrade to a new server version. To 
learn more about the {+stable-api+} feature, see the :ref:`{+stable-api+} page
<csharp-stable-api>`.

> The following code shows how you can instantiate a client that uses the
> {+stable-api+} version and a sample connection string with placeholders for
> your username and password. Instantiating the client creates a
> connection to MongoDB:

Reviewer: Issue:

1) I think the style guide mentions that when using a colon, it should introduce the content that follows it (style/punctuation). I think "Instantiating the client creates a connection to MongoDB:" is describing what happens when you instantiate a client rather than what the code example shows. I think the first sentence describes the code sample, but the second one separates it and therefore the colon is not suitable.

2) I think the conjunction is ambiguous in this sentence. One version reads as  "...code shows how you can... instantiate a client... and a sample connection string" and another as "... instantiate a client that uses the Stable API... and [uses] a sample connection string." I think the sentence structure can be changed to avoid the ambiguity.

Suggestion:

Perhaps simplifying the message could resolve both issues.

E.g.

```suggestion
The following code shows how you can specify the connection string and the Stable API client option when connecting to a MongoDB deployment:
```

Author: The code also includes a confirmation of successful connection now, so I will integrate this suggestion with that new description as well.

Connect to Atlas
~~~~~~~~~~~~~~~~

> To connect to a MongoDB deployment on Atlas, you need to create a client. You can

Reviewer: Suggestion:

No changes necessary necessary, but technically, I think you can connect without creating a client. You can use some other tool or application in which a client is already created for you. Instead, this sentence could suggest the path of action using the following language:

```suggestion
To connect to a MongoDB deployment on Atlas, create a client. You can
```
